test/integration/targets/azure_rm_aci/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+- name: Create sample container group
+  azure_rm_aci:
+    resource_group: "{{ resource_group }}"
+    name: mynewcontainergroup
+    os_type: linux
+    ip_address: public
+    ports:
+      - 80
+    containers:
+      - name: mycontainer1
+        image: httpd
+        memory: 1.5
+        ports:
+          - 80
+          - 81
+      - name: mycontainer2
+        image: httpd
+        memory: 1.5
+  register: output
+
+- name: Assert the ACI instance is well created
+  assert:
+    that:
+      - output.changed
+      - output.state.provisioning_state == 'Creating'
```

Comment:
"Creating" or "Succeeded"?

Reply:
At the moment we are not waiting for "Succeeded". We could but that would require additional code, as create_or_update leaves with "Creating" state and it's work is done.

Reply:
just checked what happens if we try to create 2 containers and one of them fails.
in such case it will stay in "Creating" state and keep retrying to start failed container.